### DISCUSSION TITLE
Which -pc_type is used in this step?

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [idaholab/moose#23899] [Q&A General] [answered]
Hello everyone!
May I ask a question? In PJFNK, the last step we get  
![1680147727689](https://user-images.githubusercontent.com/126685008/228723247-a907bd1f-8d94-4850-8e96-52763860c9f5.png)
We want to get
![1680147849645](https://user-images.githubusercontent.com/126685008/228723321-9b5c0078-7a87-4d81-a349-8e04752ddfcf.png)
Which -pc_type is used in this step?
Thank you very much !

**Comment by lindsayad** [comment, marked as answer]
The default preconditioner in serial is ILU (https://petsc.org/release/docs/manualpages/PC/PCILU/) and the default preconditioner in parallel is block jacobi (https://petsc.org/release/docs/manualpages/PC/PCBJACOBI/) with ILU on sub-blocks (individual processes)
